scripts/build/termux_setup_self_debug.sh

```diff
@@ -0,0 +1,15 @@
+termux_print_stacktrace() {
+	local func src lineno
+	echo "💥 Error occurred at line $1 in script ${BASH_SOURCE[1]}"
+	echo "🔍 Stack trace:"
+	for i in "${!FUNCNAME[@]}"; do
+		func="${FUNCNAME[$i]}"
+		src="${BASH_SOURCE[$i]}"
+		lineno="${BASH_LINENO[$((i - 1))]}"
```

Comment:
Array subscripts are already arithmetic context.
```suggestion
		lineno="${BASH_LINENO[$i - 1]}"
```